nx-X11/lib/X11/XlibInt.c

```diff
@@ -431,28 +420,57 @@ _XWaitForWritable(
 #endif
 
 #ifndef NX_TRANS_SOCKET
-#  ifdef USE_POLL
+#ifdef USE_POLL
 	    filedes.events = POLLIN;
 	filedes.events |= POLLOUT;
-#  else
+#else
 	FD_SET(dpy->fd, &r_mask);
         FD_SET(dpy->fd, &w_mask);
-#  endif
+#endif
 #endif /* #ifndef NX_TRANS_SOCKET */
 
 	do {
+#ifdef NX_TRANS_SOCKET
```

Comment:
NX code reinserted here... (okay, just as a follow-up to the first commit's comment(s).)